### ISSUE TITLE
Logistic_regression model has no example script

### ISSUE THREAD

**Issue by DaveDeCaprio** [closedloop-ai/cv19index#9]
The logistic regression model is currently just a pickle file with parameters.  It is not callable from the regular script.

**Comment by wgillett** [comment]
@DaveDeCaprio checking that this ticket still has value? If yes, would it make sense to add more args to predict.py to support running logistic regression predictions as an alternative to XGBoost?

**Comment by DaveDeCaprio** [comment]
Yes, this would be valuable and we haven't gotten a chance to get it done.
